verification/src/tests/uncle_verifier.rs

```diff
@@ -628,6 +613,6 @@ fn test_uncle_with_uncle_descendant() {
         let dummy_context = dummy_context(&shared);
         let uncle_verifier_context = UncleVerifierContext::new(&dummy_context, &epoch18);
         let verifier = UnclesVerifier::new(uncle_verifier_context, &new_block);
-        assert_eq!(verifier.verify(), Ok(()));
+        assert!(verifier.verify().is_ok());
```

Comment:
I think `assert_eq!(verifier.verify(), Ok(()))` is more friendly, 
when assert failed, will got 
left: {error_kind} right: `Ok(())`

But `assert!(verifier.verify().is_ok())` only got 
left: false right: ture

Just comment here, not required, we shall improve it later for test friendly, someday.
`assert_eq!(ret, Ok(()), "testing sth got unexpected error {}", ret)`

Reply:
Yes, I agree.

I just couldn't find a better way to fight against `rustc` about the compiling errors 😞 :

```
error[E0369]: binary operation `==` cannot be applied to type `std::result::Result<core::cell::ResolvedTransaction<'_>, ckb_error::Error>`
   --> util/types/src/core/cell.rs:707:9
    |
707 |         assert_eq!(result, Ok(()));
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |         |
    |         std::result::Result<core::cell::ResolvedTransaction<'_>, ckb_error::Error>
    |         std::result::Result<(), _>
    |
    = note: an implementation of `std::cmp::PartialEq` might be missing for `std::result::Result<core::cell::ResolvedTransaction<'_>, ckb_error::Error>`
    = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)

error: aborting due to previous error
```